src/deploy/osp_deployer/deployer.py

```diff
@@ -215,6 +225,8 @@ def deploy():
         director_vm.node_introspection()
         director_vm.update_sshd_conf()
         director_vm.assign_node_roles()
+        # Should we be calling function below?
+        # director_vm.revert_sshd_conf()?
         director_vm.revert_sshd_conf
```

Comment:
We need to temporary do this so we can run assign role against all the nodes concurrently, but yes i believe that restoring the config as it was once we're done is good practice.

Reply:
So method should be called `director_vm.revert_sshd_conf()` correct?  Will patch in separate PR